canonical/mlflow-operator
# Add 2.15 mlflow bundles

Closes: https://github.com/canonical/mlflow-operator/issues/253

- Fix bundles for latest 
- Add bundles for 2.15

I have tested locally by:
- Deploy [kubeflow](https://charmed-kubeflow.io/docs/get-started-with-charmed-kubeflow).
- Deploy MLFlow bundle (edge one from this PR)
- Deploy resource dispatcher `juju deploy resource-dispatcher --channel 2.0/stable --trust`
- Relate to dispatcher 
```
juju relate mlflow-server:pod-defaults resource-dispatcher:pod-defaults
juju relate mlflow-server:secrets resource-dispatcher:secrets
```
- I have created notebook with the MLFlow options
![image](https://github.com/user-attachments/assets/64f6c40c-eaa1-4fc5-bcd0-159464e0e8e9)
- In that notebook I have created experiment and registered a model. I can see the experiment and model in MLflow dashboard 
![image](https://github.com/user-attachments/assets/db895ddf-5e0a-477f-8364-b4588a07b1cb)
